src/main/java/edu/umich/its/lti/TcSessionCustomData.java

```diff
@@ -0,0 +1,25 @@
+package edu.umich.its.lti;
+
+import java.util.HashMap;
+
+import javax.servlet.http.HttpServletRequest;
+
+import edu.umich.its.lti.utils.OauthCredentials;
+
+public class TcSessionCustomData extends TcSessionData{
+	
```

Comment:
README.txt should be updated why this class was extending `TcSessionData` and with respect to Custom parameters